src/components/data/LocalVariablesContent.js

```diff
@@ -179,7 +213,28 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
                     value: editType,
                     onChange: ({ value }) => setEditType(value),
                     options: ['string', 'number', 'boolean', 'string list', 'number list', 'boolean list']
-                  }),
+                  })
+                ])
```

Comment:
Can we re-align as follows for consistency?

```javascript
              editIndex === rowIndex && h(Fragment, [
                h(Select, {
                  'aria-label': 'data type',
                  styles: { container: base => ({ ...base, marginLeft: '1rem', width: 150 }) },
                  isSearchable: false,
                  isClearable: false,
                  menuPortalTarget: document.getElementById('root'),
                  getOptionLabel: ({ value }) => _.startCase(value),
                  value: editType,
                  onChange: ({ value }) => setEditType(value),
                  options: ['string', 'number', 'boolean', 'string list', 'number list', 'boolean list']
                })
              ])
```

Reply:
Done.